### PR TITLE
Fixed steps under "Allow only team owners to post to the General channel" (create-an-org-wide-team.md)

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5654.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-an-org-wide-team.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ To get the most out of your org-wide team, we recommend team owners do the follo
 
 ### Allow only team owners to post to the General channel
 
-Reduce channel noise by having only team owners post to the General channel. Go to the team and click **˙˙˙ More options** > **Manage Team**. On the **Settings** tab, click **Member permissions** > select **Only owners can post messages**.
+Reduce channel noise by having only team owners post to the General channel. Go to the team, locate the General channel, and click **˙˙˙ More options** > **Manage channel**. On the **Channel settings** tab, click **Permissions** > select **Only owners can post messages**.
```
Comment (JohanFreelancer9) [inline Teams/create-an-org-wide-team.md:54]:
```suggestion
Reduce channel noise by having only team owners post to the General channel. Go to the team, locate the General channel, and click **˙˙˙ More options** > **Manage channel**. On the **Channel settings** tab, click **Permissions** and select **Only owners can post messages**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**